cmd/jaeger/internal/extension/jaegerquery/server.go

```diff
@@ -122,6 +129,25 @@ func (s *server) addArchiveStorage(opts *querysvc.QueryServiceOptions, host comp
 	return nil
 }
 
+func (s *server) addMetricStorage(opts *querysvc.QueryServiceOptions, host component.Host) (metricsstore.Reader, error) {
+	if s.config.MetricStorage == "" {
+		s.telset.Logger.Info("Metric storage not configured")
+		return nil, nil
```

Comment:
should this return `disabled.NewMetricsReader()`?

Reply:
yes